apps/website/src/pages/docs/colorpicker.mdx

```diff
@@ -14,16 +14,66 @@ import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-colorpicker' />
-
 <LiveExample src='ColorPicker.main.tsx'>
   <AllExamples.ColorPickerMainExample client:load />
 </LiveExample>
 
-The color picker is extremely flexible. Bits and pieces can be used based on product requirements. If certain parts are not required for your context, they can be left out.
+Color pickers allow users to browse and apply a color to a selected item. The color picker is extremely flexible. Bits and pieces can be used based on product requirements. If certain parts are not required for your context, they can be left out.
```

Comment:
This section is written twice: once here and then again in the Subcomponent section

Reply:
Removed